types/json/binary_functions.go

```diff
@@ -66,19 +66,21 @@ func (bj BinaryJSON) Unquote() (string, error) {
 	switch bj.TypeCode {
 	case TypeCodeString:
 		tmp := string(hack.String(bj.GetString()))
-		s, err := unquoteString(tmp)
-		if err != nil {
-			return "", errors.Trace(err)
+		tlen := len(tmp)
+		if tlen < 2 {
+			return tmp, nil
 		}
-		// Remove prefix and suffix '"'.
-		slen := len(s)
-		if slen > 1 {
-			head, tail := s[0], s[slen-1]
-			if head == '"' && tail == '"' {
-				return s[1 : slen-1], nil
+		head, tail := tmp[0], tmp[tlen-1]
+		if head == '"' && tail == '"' {
+			// Remove prefix and suffix '"'.
+			s, err := unquoteString(tmp[1 : tlen-1])
+			if err != nil {
+				return s, errors.Trace(err)
```

Comment:
I think we can remove the `Trace`.

Reply:
The error is returned by `unquoteString`, so I think it is better to use `Trace`? @lonng

Reply:
@bb7133 All error returned by `unquoteString` have been wrapt with`Trace`. (anyway, not a strong opinion)

Reply:
I see, thanks.

Reply:
Addressed